chi/network.py

```diff
@@ -703,34 +712,47 @@ def nuke_network(network_name):
     Args:
         network_name (str): The network name.
     """
+    network = get_network(network_name)
     network_id = get_network_id(network_name)
```

Comment:
If we fetch the network we don't also need to fetch the ID again, it can just be `network_id = network["id"]`

Reply:
Nice. Noted, and incorporated.